content/docs/aws/deploy/uninstall-kubeflow.md

```diff
@@ -10,6 +10,9 @@ weight = 10
 ```
 cd ${KFAPP}
 kfctl delete all -V
+
+# TODO(@Jeffwan): move this line once #3767 is resolved
```

Comment:
I think this comment shouldn't be visible in the doc. Rather than putting the comment inside the code snippet, rather add it as an HTML comment in the doc. (Actually, it'd be better not to have a TODO at all, but rather track this change in an issue.)

Reply:
Agree. I will remove this comment.